common/include/opae/properties.h

```diff
@@ -495,7 +495,7 @@ fpga_result fpgaPropertiesSetModel(fpga_properties prop,
  * @note This API is not currently supported.
  */
 fpga_result fpgaPropertiesGetLocalMemorySize(const fpga_properties prop,
-					     uint64_t *lms);
+		uint64_t *lms);
```

Comment:
Unintentional change of indentation? (more below)

Reply:
Not unintentional.  I'm using AStyle to format my source changes, which will automate formatting and make things consistent.  We should discuss at some time, because manually "fixing" all the formatting cruft is extremely time-consuming.